Harvest.Web/Controllers/TempInvoiceController.cs

```diff
@@ -12,13 +12,13 @@
 namespace Harvest.Web.Controllers
 {
     [Authorize]
-    public class InvoiceController : SuperController
+    public class TempInvoiceController : SuperController
     {
         private readonly AppDbContext _dbContext;
         private readonly IUserService _userService;
         private readonly IInvoiceService _invoiceService;
 
-        public InvoiceController(AppDbContext dbContext, IUserService userService, IInvoiceService invoiceService)
+        public TempInvoiceController(AppDbContext dbContext, IUserService userService, IInvoiceService invoiceService)
```

Comment:
@sprucely The plan is to replace this when the other methods like unbilled are moved into react?

Reply:
Should have said remove, not replace.

Reply:
Not sure. I was just getting it out of the way.